tests/test_this_is_setup_properly.py

```diff
@@ -1,2 +0,0 @@
-def test_for_testing_sake():
-    assert 1 == 1
```

Comment:
Dunno, I removed it. Think is was the original test from the empty repo to prove pytest worked